src/main/starlark/core/repositories/initialize.release.bzl

```
@@ -96,7 +96,7 @@ def kotlin_repositories(
         if (criteria and compiler_release.version.startswith(criteria.prefix)) or (not selected_version and not criteria):
             selected_version = version
 
-    if configured_repository_name:  # without a repository name, no default kt_* rules repository is created.
+    if configured_repository_name != "":  # without a repository name, no default kt_* rules repository is created.
```

Uuuuuh.... is this change in starlark?

---

Hmm the build failed with:
```
Caused by: java.lang.ClassCastException: class net.starlark.java.eval.NoneType cannot be cast to class java.lang.String (net.starlark.java.eval.NoneType is in unnamed module of loader 'app'; java.lang.String is in module java.base of loader 'bootstrap')
```

I think we need this instead?
```suggestion
    if configured_repository_name and configured_repository_name != "":  # without a repository name, no default kt_* rules repository is created.
```